code/style.css

```diff
@@ -1,5 +1,360 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+html,
+body {
+  height: 100%;
+  font-family: helvetica, arial, sans-serif;
+  font-weight: 200;
+}
+
+header {
+  background-image: url(/img/header.jpg);
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  color: white;
+  width: 100%;
+  height: 100%;
+  min-height: 300px;
+  max-height: 500px;
+  box-sizing: border-box;
+}
+
+h1 {
+  font-size: 50px;
+  font-weight: 300;
+  margin: 0;
+}
+
+h2 {
+  font-weight: 100;
+  font-size: 30px;
+}
+
+p {
+  color: black;
```

Comment:
You don't need to specify black color. It's usually default, unless you've set a color to the parent element. 😊